docs/manual/common/guide/devmode/CassandraServer.md

```diff
@@ -44,11 +44,11 @@ In sbt:
 
 @[cassandra-clean-on-start](code/build-cassandra-opts.sbt)
 
-# Keyspace
+## Keyspace _(deprecated)_
```

Comment:
Sorry, I don't really understand. This entire page is about configuring dev mode.

The general introduction of Cassandra keyspaces and why they must be configured in each service has moved to the `PersistentEntityCassandra.md` page below.